### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by mrlioncub**
New working file for Docker Hub 

* update Dockerfile
* update README.md

working container https://hub.docker.com/r/mrlioncub/hamara

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,15 +42,29 @@ Flags:
   -k, --key string    API key with Admin rights from Grafana
 ```
 
+or docker run:
+
+```bash
+docker run --rm mrlioncub/hamara
```
Comment (jorgelbg) [inline README.md:48]: nit: Now that we have the `trivago/hamara` image, we should use it.

```suggestion
docker run --rm trivago/hamara
```

**Review by jorgelbg — CHANGES_REQUESTED** [review]
Hi @mrlioncub thank you very much for your contribution 🎉! This PR definitively improves the Docker setup for this project.

I left a few small comments, mostly around using `trivago/hamara` for the Docker image (instead of the fork) in the badges/documentation.

> I also think that we should automatically build & push a new Docker image every time that a new release is tagged, but we can track this in a different issue/PR.

**Review by mrlioncub — COMMENTED** [review]
Everything is fine, but we need to change README.md:
mrlioncub/hamara -> trivago/hamara

**Review by jorgelbg — APPROVED** [review]
Great work @mrlioncub, I will go ahead and merge this. I think we're in good shape now to build a docker image as part of the release process when a new version is tagged.

**Pull request merged**